galsim/config/input_powerspectrum.py

```diff
@@ -70,6 +71,7 @@ def getKwargs(self, config, base, logger):
             rs = base['image']['random_seed']
             if not isinstance(rs, list): rs = [rs]
             first_seed += 31415  # An arbitrary offset to avoid the chance of unwanted correlations
+            first_seed = BaseDeviate(first_seed).raw()
```

Comment:
This hard-coded value seems dangerous. Do we need it now that we redraw things? Maybe rewdraw things twice?

Reply:
By rewdraw twice I mean, `BaseDeviate(BaseDeviate(first_seed).raw()).raw()` or something.

Reply:
I think now that we are subsequently applying raw() to get the final value, even +=1 is fine.  Before, I wanted it big enough that there wasn't some weird chance of the shear power spectrum realization coupling to the potentially random locations of galaxies in an image.  Probably was me being overly paranoid to even make a second rng here in the first place.  But now I think any offset value (1 or 31415 or anything but 0) is functionally equivalent.